src/uct/sm/knem/knem_md.c

```diff
@@ -359,7 +359,7 @@ static ucs_status_t uct_knem_md_open(const char *md_name,
                 uct_knem_md_close(&knem_md->super);
                 return status;
             } else {
-                ucs_debug("Could not create registration cache for: %s",
+                ucs_debug("Could not create registration cache: %s",
```

Comment:
could...

Reply:
i will merge it for now, maybe will make some script to fix all messages in the code..